frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -16,91 +16,60 @@ export interface EmptySProps {
-            <CardFooter>
-              <Link
-                className="pf-c-button pf-m-secondary"
-                to={`/deploy-image?preselected-ns=${activeNamespace}`}
-              >
-                Deploy Image
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Browse Catalog</CardHeader>
-            <CardBody>Browse the catalog to discover, deploy and connect to services</CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/catalog">
-                Browse Catalog
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Import from Dockerfile</CardHeader>
-            <CardBody>Import your Dockerfile from your git repo to be built & deployed</CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/import?importType=docker">
-                Import from Dockerfile
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Import YAML</CardHeader>
-            <CardBody>Create or replace resources from their YAML or JSON definitions.</CardBody>
-            <CardFooter>
-              <Link
-                className="pf-c-button pf-m-secondary"
-                to={formatNamespacedRouteForResource('import', activeNamespace)}
-              >
-                Import YAML
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Add Database</CardHeader>
-            <CardBody>
-              Browse the catalog to discover database services to add to your application
-            </CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/catalog?category=databases">
-                Add Database
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-      </Grid>
-    </div>
-  </React.Fragment>
-);
+const ODCEmptyState: React.FC<Props> = ({ title, activeNamespace }) => {
+  const navigateTo = (url: string) => {
+    history.push(url);
+  };
+
+  return (
+    <React.Fragment>
+      <div className="odc-empty-state__title">
+        <PageHeading title={title} />
+        <p className="co-catalog-page__description">
+          Select a way to create an application, component or service from one of the options.
+        </p>
+      </div>
+      <div className="odc-empty-state__content">
+        <CatalogTile
+          onClick={() => navigateTo('/import?importType=git')}
```

Comment:
You should be using an `href` so that it is a proper link and not `onClick`

Reply:
@christianvogt When I use ```href``` then the whole page reload while navigating.

Reply:
Looked at the component and it seems to not support switching the link component to render a react-router link.
So using `onClick` is ok. But you should still supply an href so that it can be right-clicked -> opened.
Make sure that the `onClick` prevents default to ensure that the page doesn't reload.